src/decode/rangecoder.rs

```diff
@@ -180,6 +180,11 @@ impl BitTree {
     ) -> io::Result<u32> {
         rangecoder.parse_reverse_bit_tree(self.num_bits, self.probs.as_mut_slice(), 0, update)
     }
+
+    #[inline]
```

Comment:
This shouldn't be necessary. If this really brings a performance improvement (assuming the compiler didn't already inline it by default), let's do that in a separate PR with proper before/after benchmarks.